src/app/modules/admin-core/services/private-guard/private.guard.ts

```diff
@@ -22,7 +22,7 @@ export class PrivateGuard implements CanActivate {
     }
     allowedArray = this.keyCloakService.getAllowedUrls();
 
-    if (allowedArray.includes(this.url)) {
+    if (localStorage.getItem('user_role') === 'OBS_REVIEWER') {
```

Comment:
Why is the private guard checking for  roles?

Reply:
i am not using any private or public guard cuurently, Now i am using only route guard